### PR TITLE
Send email notification

### PR DESCRIPTION
**Pull request by jason-tung**
### Summary 

Clicking heart or bell sends an email to the associated person from the person list. The email at the moment is very simple:
```
subject: `You got a ${variant}!`,
text: `${senderName} from ${group.name} is sending you a ${variant}!`,
```
We will need to place a Sendgrid key in `functions/src/sendgrid-key.json` and change the email in [`functions/src/send-email.ts`](https://github.com/cornell-dti/samwise/blob/e25835094e61b973ec14b765aedb87cb6cbdf460/functions/src/send-email.ts#L46). 

### Test Plan 

Playtested sending email to myself and peter.
Fiddle with it on staging, I suppose.

### CODE REVIEW

**Comment by jason-tung** [comment]
Linter warnings are because there is no API key since it's gitignored

File: frontend/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.12.0",
     "@fortawesome/react-fontawesome": "^0.1.8",
     "common": "0.1.0",
+    "cors": "^2.8.5",
```
Comment (SamChou19815) [inline frontend/package.json:18]: We shouldn't need this

File: frontend/src/components/GroupView/MiddleBar/GroupViewMiddleBarMemberRow.tsx [hunk 2]
```diff
@@ -1,16 +1,31 @@
 import React, { ReactElement } from 'react';
 import { faPlus } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import firebase from 'firebase';
```
Comment (SamChou19815) [inline frontend/src/components/GroupView/MiddleBar/GroupViewMiddleBarMemberRow.tsx:4]: Import from firebase/app instead

File: functions/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+sendgrid.env
```
Comment (SamChou19815) [inline functions/.gitignore:1]: Not sure why you need this. Looks like you already gitignored the token json

File: functions/src/sendgrid-key.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"SENDGRID_API_KEY":""}
```
Comment (SamChou19815) [inline functions/src/sendgrid-key.json:1]: this should not be here

**Review by SamChou19815 — COMMENTED** [review]
@jason-tung could you send me an email using the feature you built, so that I know it works?

File: functions/src/sendgrid-key.json (hunk 4 above)
Comment (jason-tung) [reply functions/src/sendgrid-key.json]: Is my CI code incorrect? I removed the file and the CI command to write a dummy file is still there but the CI is failing

Comment (SamChou19815) [reply functions/src/sendgrid-key.json]: @jason-tung you need to add the CI command to all the jobs that might need the key. It's left as an exercise for the reader to figure out what are those CI jobs 😉

Comment (jason-tung) [reply functions/src/sendgrid-key.json]: 🤔

**Review by SamChou19815 — APPROVED** [review]

**Pull request merged**